substrate/frame/executive/src/tests.rs

```diff
@@ -1323,68 +1323,6 @@ fn try_execute_tx_forbidden_errors() {
 	});
 }
 
-/// Check that `ensure_inherents_are_first` reports the correct indices.
-#[test]
-fn ensure_inherents_are_first_works() {
```

Comment:
But do we have a test for this now? maybe one test that ensures that if inherents are not first then it fails.